### ISSUE TITLE
config.json example?

### ISSUE THREAD

**Issue by msdart** [thomasperge/RPG-Discord-Bot#2]
node:internal/modules/cjs/loader:1080
  throw err;
  ^

Error: Cannot find module './config.json'
Require stack:
- C:\Users\vsemushenkov\Desktop\RPG-Discord-Bot-main\main.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)
    at Module._load (node:internal/modules/cjs/loader:922:27)
    at Module.require (node:internal/modules/cjs/loader:1143:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (C:\Users\vsemushenkov\Desktop\RPG-Discord-Bot-main\main.js:4:16)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
    at Module.load (node:internal/modules/cjs/loader:1119:32)
    at Module._load (node:internal/modules/cjs/loader:960:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  code: 'MODULE_NOT_FOUND',
  
  
  config.json.example pls?

**Comment by thomasperge** [comment]
### Sorry, I forget the gitexemple:

`prefix` = `"!"`  # the prefix of the bot (type string)
`token` = `"your_access_token"`  # access token of your bot
`mongodb` = `"your_MongoDB_access_link"`  # you should create a MongoDB account and create a database

**Comment by msdart** [comment]
Yes thanks, but my bot doesn't send anything in response to !rhelp / rhelp etc
RPG Bot: Ready !
MongoDB : Ready !
